docs/installation/binary_installation.md

```diff
@@ -0,0 +1,82 @@
+# Binary Installation
+
+The packages for analyzing trace data with CARET can also be installed in binary.
+
+<prettier-ignore-start>
+!!! Note
+    Only the packages for analyzing trace data can be installed by binary installation.<br>
+    The binary does not include the packages for recording trace data.
+<prettier-ignore-end>
+
```

Comment:
I think it is better to put this note under the table.

Reply:
Ok, fixed in https://github.com/tier4/caret_doc/pull/335/commits/f123f0bce7bf4f20e2f9dcf33ea016664c2fb473.